tests/benchmark/data/hnsw_indices/ hnsw_indices_updated.txt

```diff
@@ -1,2 +1,3 @@
-https://dev.cto.redis.s3.amazonaws.com/VectorSimilarity/DBpedia-n500K-cosine-d768-M65-EFC512.hnsw
-https://dev.cto.redis.s3.amazonaws.com/VectorSimilarity/DBpedia-n500K-cosine-d768-M65-EFC512-updated.hnsw
+https://dev.cto.redis.s3.amazonaws.com/VectorSimilarity/dbpedia-cosine-dim768-M65-efc512-n500k.hnsw_v3
+https://dev.cto.redis.s3.amazonaws.com/VectorSimilarity/dbpedia-cosine-dim768-M65-efc512-n500k-updated.hnsw_v3
+https://dev.cto.redis.s3.amazonaws.com/VectorSimilarity/dbpedia-cosine-dim768-test_vectors.raw
```

Comment:
Is there a reason that we need the test vectors here now even though they weren't here before?

Reply:
It makes more sense to have it here if one wishes to download these datasets only. On the `*_all.txt` it appears only once.